client/routes/workflow/history.vue

```diff
@@ -367,17 +368,16 @@ export default {
         </div>
       </div>
       <div class="actions">
-        <a href="#" @click.prevent="toggleShowDagGraph()"
-          >{{ this.graphView === GRAPH_VIEW_DAG ? 'hide' : 'show' }} graph</a
```

Comment:
nit: just curious - why you are omitting 'this' ?

Reply:
Good question. Vue supports both, similar to angular in that it does not need `this` context as `this` is assumed.
See https://eslint.vuejs.org/rules/this-in-template.html for more details (should actually turn this linter on)